specification/network/resource-manager/Microsoft.Network/stable/2023-11-01/networkWatcher.json

```diff
@@ -2812,15 +2821,48 @@
       "properties": {
         "storageId": {
           "type": "string",
-          "description": "The ID of the storage account to save the packet capture session. Required if no local file path is provided."
+          "description": "The ID of the storage account to save the packet capture session. Required if no localPath or filePath is provided."
         },
         "storagePath": {
           "type": "string",
           "description": "The URI of the storage path to save the packet capture. Must be a well-formed URI describing the location to save the packet capture."
         },
         "filePath": {
           "type": "string",
-          "description": "A valid local path on the targeting VM. Must include the name of the capture file (*.cap). For linux virtual machine it must start with /var/captures. Required if no storage ID is provided, otherwise optional."
+          "description": "This path is invalid if 'Continuous Capture' is provided with 'true' or 'false'. A valid local path on the targeting VM. Must include the name of the capture file (*.cap). For linux virtual machine it must start with /var/captures. Required if no storage ID is provided, otherwise optional."
```

Comment:
For Continuous Capture, should we use the parameter name "continuousCapture"

Reply:
it just a description, like storage id.